book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] for the association to work Getters and Setters for @rentals should be present